### PR TITLE
Add various templates for blog pages

### LINKED ISSUE #11: Proceed with the Django Girls tutorial
Do a PR per each tutorial page.

### PR DESCRIPTION
**Pull request by makkaronis4e**
Add templates with post list, detailed post and post creation form.
Resolve #11

### CODE REVIEW

**Comment by makkaronis4e** [comment]
@webknjaz NEW PR

File: blog/forms.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""Post form."""
```
Comment (webknjaz) [inline blog/forms.py]: No, it's a generic module, it's not dedicated to the single form class.

File: blog/forms.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from django import forms
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Name of post form."""
```
Comment (webknjaz) [inline blog/forms.py]: WAT?

File: blog/forms.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    class Meta:
+        """Show model which would be used for form."""
```
Comment (webknjaz) [inline blog/forms.py]: It's not an action. It's a config/metadata, so don't use imperative mood.

File: blog/templates/blog/base.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        {% if user.is_authenticated %}
+        <a href="{% url 'post_new' %}" class="top-menu">
+            <span class="glyphicon glyphicon-plus"></span>
+        </a>
+        {% endif %}
+        <span class="glyphicon glyphicon-plus">
+        </span>
+        </a>
+        <h1>
+            <a href="/">Django Girls Blog</a>
```
Comment (webknjaz) [inline blog/templates/blog/base.html]: Plz use url lookup here + add i18n tag

File: blog/templates/blog/post_edit.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{% extends 'blog/base.html' %}
+
+{% block content %}
+    <h1>New post</h1>
+    <form method="POST" class="post-form">{% csrf_token %}
+        {{ form.as_p }}
+        <button type="submit" class="save btn btn-default">Save</button>
```
Comment (webknjaz) [inline blog/templates/blog/post_edit.html]: Plz use i18n

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Comment by makkaronis4e** [comment]
@webknjaz add changes

File: blog/forms.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+"""Generic module for post form."""
```
Comment (webknjaz) [inline blog/forms.py]: No, it's not for post form. Just forms declarations.

File: blog/templates/blog/post_edit.html [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{% extends 'blog/base.html' %}
+
+{% block content %}
+    <h1>New post</h1>
```
Comment (webknjaz) [inline blog/templates/blog/post_edit.html]: i18n

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Comment by makkaronis4e** [comment]
@webknjaz fixed

File: blog/forms.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
+"""Generic module."""
```
Comment (webknjaz) [inline blog/forms.py]: Okay, write: `Forms setup.`

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Review by webknjaz — APPROVED** [review]

**Pull request merged**